frontend/src/components/Question.vue

```diff
@@ -43,45 +43,43 @@ const components = getComponents();
 
 const props = defineProps<QuestionWrapperProps>();
 
-const QuestionSpecComponent: ComputedRef = computed(() => {
+const QuestionSpec: ComputedRef = computed(() => {
     if (props.question.kind in components) {
         return components[props.question.kind];
     } else {
         throw Error(`Unknown QuestionKind "${props.question.kind}"`);
     }
 });
 
-const difficultiesColors: Record<Quizoot.Difficulty, string> = {
+const DIFFICULTY_COLOR_MAP: Record<Quizoot.Difficulty, string> = {
     EASY: 'green',
     MEDIUM: 'orange',
-    HARD: 'red',
+    HARD: 'var(--palette-well-read)',
 };
 
-const questionDifficultyColor: ComputedRef = computed(
-    () => difficultiesColors[props.question.difficulty]
+const difficultyColor: ComputedRef = computed(
+    () => DIFFICULTY_COLOR_MAP[props.question.difficulty]
 );
 
-const hintIsDisplayed: Ref<boolean> = ref(false);
+const displayHint: Ref<boolean> = ref(false);
 
 function toggleHint() {
-    hintIsDisplayed.value = !hintIsDisplayed.value;
+    displayHint.value = !displayHint.value;
 }
 
-const hintStyleProps: ComputedRef = computed(() => {
```

Comment:
I removed this because there is a better way : you can bind the class name dynamically. It's better because you avoid mixing script and styles. 

```typescript
const displayHint = ref(false);

<div class="static classes" :class ="{ display: displayHint, 'text-danger': anotherBoolVariable}"></div>

See https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes

Reply:
Yes, I knew, but on the go I prefered the previous way. Anyway let's use class binding

Reply:
Ok. I honestly prefer to keep css styles as much as i can away from script. Imagine you want to add more styles in the future, you'll just add props in the JS object and your script get longer for no reason.

Reply:
you got a point, indeed 😇